code/datums/wires/airlock.dm

```diff
@@ -84,6 +94,12 @@
 	status += "The timer is powered [A.autoclose ? "on" : "off"]."
 	status += "The speed light is [A.normalspeed ? "on" : "off"]."
 	status += "The emergency light is [A.emergency ? "on" : "off"]."
+
+	if(A.unres_sensor)
+		status += "The unrestricted exit display is [A.unres_sides ? "indicating that it is letting people pass from the [dir2text(REVERSE_DIR(A.unres_sides))]" : "faintly flickering"]."
+	else
+		status += "The unrestricted exit display is completely inactive."
```

Comment:
Do we care about showing this if it isn't there?

Reply:
I thought it would look much more apparent if a door had a sensor or not, I'm not terribly strong on keeping it but I personally prefer the verbosity on absolutely knowing if the functionality is there at all rather than have to wig out over duds.